src/components/chips/js/chipDirective.js

```diff
@@ -0,0 +1,54 @@
+(function () {
+  'use strict';
+  angular
+      .module('material.components.chips')
+      .directive('mdChip', MdChip);
+
+  /**
+   * @ngdoc directive
+   * @name mdChip
+   * @module material.components.chips
+   *
+   * @description
+   * `<md-chip>` is a component used within `<md-chips>` and is responsible for rendering individual
+   * chips.
+   *
+   *
+   * @usage
+   * <hljs lang="html">
+   *   <md-chip>{{$chip}}</md-chip>
+   * </hljs>
+   *
+   */
+
+  // This hint text is hidden within a chip but used by screen readers to
+  // inform the user how they can interact with a chip.
+  var DELETE_HINT_TEMPLATE = '\
+      <span ng-if="!$mdChipsCtrl.readonly" class="visually-hidden">\
+        {{$mdChipsCtrl.deleteHint || $mdChipsCtrl.defaultDeleteHint}}\
```

Comment:
Instead of putting an `||` in the binding expression, just default `deleteHint` to the default message.